main.go

```diff
@@ -28,14 +60,21 @@ func main() {
 	// 3. Создание контекста
 	// ...
 
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Можно так же использовать контекст с отменой
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
	defer cancel()
```